src/coreComponents/common/TypeDispatch.hpp

```diff
@@ -375,17 +375,25 @@ bool dispatch( LIST const combinations,
 
   if( !success )
   {
-    auto typePrinter = []( auto t ){ return LvArray::system::demangle( typeid( typename decltype(t)::type ).name() ); };
-    auto typeListPrinter = [typePrinter]( auto tlist ){ return internal::listToString( typename decltype( tlist )::type{}, "\n  ", "", typePrinter ); };
+    // auto typePrinter = []( auto t ){ return LvArray::system::demangle( typeid( typename decltype(t)::type ).name() ); };
+    // auto typeListPrinter = [typePrinter]( auto tlist ){ return internal::listToString( typename decltype( tlist )::type{}, "\n  ", "", typePrinter ); };
 
     GEOS_ERROR( "Types were not dispatched. The types of the input objects are:\n" <<
-                "( "<<(  ( "\n  " + LvArray::system::demangle( internal::typeIdWrapper( objects ).name() ) ) + ... )<<" \n)\n"<<
-                "and the dispatch options are:\n"<<
-                internal::listToString( combinations, "\n(", "\n)", typeListPrinter ) );
+                "( "<<(  ( "\n  " + LvArray::system::demangle( internal::typeIdWrapper( objects ).name() ) ) + ... )<<" \n)\n"
+                // <<
+                // "and the dispatch options are:\n"<<
+                // internal::listToString( combinations, "\n(", "\n)", typeListPrinter )
```

Comment:
Because `combinations` has so many different types I have run into compiler bugs when trying to print it out. Luckily the stack trace here seems to work find and that contains the types.